zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -2126,6 +2126,17 @@ options are used to configure the [AdminServer](#sc_adminserver).
   The time interval for rate limiting snapshot command to protect the server.
   Defaults to 5 mins.
 
+* *admin.restore.enabled* :
+  (Java system property: **zookeeper.admin.restore.enabled**)
+  The flag for enabling the restore command. Defaults to false.
+  It will be enabled by default once the auth support for admin server commands
+  is available.
+
+* *admin.restore.intervalInMS* :
+  (Java system property: **zookeeper.admin.restore.intervalInMS**)
+  The time interval for rate limiting restore command to protect the server.
+  Defaults to 5 mins.
```

Comment:
Do we need this setting separately for all admin commands?
Shouldn't we just introduce a new general setting for the admin interface which would rate limit all admin requests coming in? 
I don't think anybody would like to set this individually for commands.

Reply:
I don't see a real need to configure the interval differently for the snapshot and restore command at this point.

Yes, we can keep it simple and have just one general setting for all the admin commands that need to be rate limited.

Reply:
updated